.github/workflows/pull-upstream.yml

```
@@ -11,10 +11,10 @@ jobs:
     timeout-minutes: 15
 
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v3.3.0
```

```suggestion
    - uses: actions/checkout@v3
```
Set only major and will always run the latest